<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteUserScreen</class>
 <widget class="QWidget" name="DeleteUserScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#DeleteUserScreen {
    background-color: #303030;
}
QLabel {
    color: white;
    font-size: 48pt;
}
QListWidget {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 20px;
    font-size: 48pt;
}
QListWidget::item {
    padding: 20px;
    min-height: 80px;
}
QListWidget::item:selected {
    background-color: #505050;
}
QListWidget::item:hover {
    background-color: #454545;
}
QPushButton {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 10px;
    font-size: 48pt;
    min-width: 300px;
    min-height: 120px;
}
QPushButton:hover {
    background-color: #505050;
    border-color: #606060;
}
QPushButton:pressed {
    background-color: #606060;
}
QPushButton#deleteButton {
    background-color: #803030;
    border-color: #905050;
}
QPushButton#deleteButton:hover {
    background-color: #904040;
    border-color: #a06060;
}
QPushButton#deleteButton:pressed {
    background-color: #a05050;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>40</number>
   </property>
   <property name="leftMargin">
    <number>80</number>
   </property>
   <property name="topMargin">
    <number>80</number>
   </property>
   <property name="rightMargin">
    <number>80</number>
   </property>
   <property name="bottomMargin">
    <number>80</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Delete User</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="userListWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
